google/earthenterprise
# EE44 - Moving Development Content

https://github.com/google/earthenterprise/issues/44
Moved the development content (build instructions) off the opengee page and provided a link to the main GitHub page in its place. All the content is already in the Github wiki (and up to date) so any user can quickly navigate and find it there.

Also, added a small fix so that when the Develop tab is selected on opengee the "Develop" tab is highlighted. This previously always highlighted Documentation.

To verify this change
1. Go to: https://tst-ahernandez.github.io/earthenterprise/index.html
2. Navigate to the Develop tab
3. Verify that the page content is updated with all the build instructions removed
